src/js/electron/quitter.ts

```diff
@@ -21,8 +22,13 @@ export function handleQuit(manager: $WindowManager, store, session, zqd) {
     }
   })
 
-  app.on("quit", () => {
-    zqd.close()
+  app.on("quit", async () => {
+    try {
+      await zqd.close()
```

Comment:
Move this `await zqd.close()` to after manager.quit() and before app.quit() in the "before-quit" handler.